### ISSUE TITLE
Channel Tabs no longer working

### ISSUE THREAD

**Issue by zany130** [RobinRMC/VencordPlus#41]
### Discord Username

zany130

### What happens when the bug or crash occurs?

channel tabs plugin does not load

### What is the expected behaviour?

channel tabs plugin loads and adds tabs to the discord interface

### How do you recreate this bug or crash?

1. enable channel tabs in plugins settings
2. restart discord
3. notice it does not add the tabs

### Errors

```
Logger.ts:40  Vencord+   PluginManager  Starting plugins (stage Init)
Logger.ts:40  Vencord+   PluginManager  Starting plugin NoTrack
Logger.ts:40  Vencord+   PluginManager  Starting plugin PreviewMessage
VCDRenderer:1 read if cute :3
updater.tsx:45 Error: Command failed: git remote get-url origin
/bin/sh: line 1: git: command not found

sk @ updater.tsx:45
 [FAST CONNECT] wss://gateway.discord.gg/?encoding=json&v=9&compress=zlib-stream, encoding: json, version: 9
5Unrecognized Content-Security-Policy directive 'prefetch-src'.
Logger.ts:40  Vencord+   MessageLoggerEnhanced  No logged messages in DataStore
 [FAST CONNECT] connected in 164ms
Logger.ts:40  Vencord+   WebpackInterceptor  Found Webpack module factory web.cf975c06a65832e459b4.js
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by BetterBios had no effect (Module id is 166584): /(?<=.useState\(null\),)([A-Za-z_$][\w$]*=[A-Za-z_$][\w$]*\.useMemo.{0,300})(return null!=[A-Za-z_$][\w$]*\?([A-Za-z_$][\w$]*)\.slice.{0,30}[A-Za-z_$][\w$]*,[A-Za-z_$][\w$]*\])\)/
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Undoing patch group .Messages.VIEW_ALL_ROLES, by BetterBios because replacement /(?<=.useState\(null\),)([A-Za-z_$][\w$]*=[A-Za-z_$][\w$]*\.useMemo.{0,300})(return null!=[A-Za-z_$][\w$]*\?([A-Za-z_$][\w$]*)\.slice.{0,30}[A-Za-z_$][\w$]*,[A-Za-z_$][\w$]*\])\)/ had no effect
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by FriendsSince had no effect (Module id is 269982): /,{userId:([^,]+?)}\)/
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by BetterBios errored (Module id is 420654): /(?<=\(\),\[[A-Za-z_$][\w$]*,[A-Za-z_$][\w$]*\]=[A-Za-z_$][\w$]*\.useState\(!1\),)(.{0,200})className:([A-Za-z_$][\w$]*.{0,50}.maxBioHeight\))(.{0,300})onClick:\(\)=>{.{0,300}}\)}/
 SyntaxError: Unexpected end of input
    at eval (<anonymous>)
    at Ole (patchWebpack.ts:304:31)
    at Function.set (patchWebpack.ts:80:9)
    at web.cf975c06a65832e459b4.js:12:12983563
    at web.cf975c06a65832e459b4.js:12:13029599
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by CustomEffects had no effect (Module id is 621853): /getUserProfile\((\w+)\){return (\w+)\[\w+\]}/
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by BetterActivities had no effect (Module id is 788307): /null!=([A-Za-z_$][\w$]*)&&[A-Za-z_$][\w$]*.some\([A-Za-z_$][\w$]*=>\(0,[A-Za-z_$][\w$]*.[A-Za-z_$][\w$]*\)\([A-Za-z_$][\w$]*,[A-Za-z_$][\w$]*\)\)\?/
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by UserVoiceShow had no effect (Module id is 895697): /(?<=,)(?=![A-Za-z_$][\w$]*&&![A-Za-z_$][\w$]*&&.{0,50}setNote:)/
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Main Webpack found in web.cf975c06a65832e459b4.js, initializing internal references to WebpackRequire
Logger.ts:40  Vencord+   WebpackInterceptor  Patching webpackChunkdiscord_app.push
web.cf975c06a65832e459b4.js:12 [ProcessUtilsElectron] setMemoryInformation not available.
web.cf975c06a65832e459b4.js:12 [default] [BUILD INFO] Release Channel: stable, Build Number: 329965, Version Hash: 53cd0a639784c2a4e4fa580396b639d2e0cf1488
web.cf975c06a65832e459b4.js:12 AppCrashedFatalReport: getLastCrash not supported.
Logger.ts:40  Vencord+   NoTrack  Disabling Sentry by erroring its WebpackInstance
noTrack.ts:113 Uncaught Error: Sentry successfully disabled
    at Function.set (noTrack.ts:118:23)
    at sentry.e7a4f258477a837f607a.js:453:37896
    at sentry.e7a4f258477a837f607a.js:453:38435
Logger.ts:40  Vencord+   PluginManager  Starting plugins (stage DOMContentLoaded)
showURLs.theme.css:1 
        
        
       Failed to load resource: net::ERR_FILE_NOT_FOUND
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by Homepage had no effect (Module id is 896797): /([fallbackRoute|defaultRoute]\(\){return [A-Za-z_$][\w$]*\.Routes\.)ME/g
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by BetterJoinedDate had no effect (Module id is 348540): /\(0,[A-Za-z_$][\w$]*.[A-Za-z_$][\w$]*\)\(([A-Za-z_$][\w$]*\.[A-Za-z_$][\w$]*\.extractTimestamp\([A-Za-z_$][\w$]*\)),[A-Za-z_$][\w$]*\)/
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by BetterJoinedDate had no effect (Module id is 348540): /\(0,[A-Za-z_$][\w$]*.[A-Za-z_$][\w$]*\)\(([A-Za-z_$][\w$]*\.joinedAt),[A-Za-z_$][\w$]*\)/
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by ChannelTabs had no effect (Module id is 176299): /(\?void 0:([A-Za-z_$][\w$]*)\.channelId.{0,120})[A-Za-z_$][\w$]*\.Fragment,{/
_log @ Logger.ts:40
Logger.ts:40  Vencord+   WebpackInterceptor  Patch by Homepage had no effect (Module id is 457094): /path:\[.{0,500}Routes\.MESSAGE_REQUESTS,/
_log @ Logger.ts:40
web.cf975c06a65832e459b4.js:12 [DEPRECATED] Please use `subscribeWithSelector` middleware
a @ web.cf975c06a65832e459b4.js:12
web.cf975c06a65832e459b4.js:12 [GatewaySocket] [CONNECT] wss://gateway.discord.gg, encoding: json, version: 9, compression: zlib-stream
web.cf975c06a65832e459b4.js:12 [GatewaySocket] [FAST CONNECT] successfully took over websocket, state: Object
web.cf975c06a65832e459b4.js:12 [GatewaySocket] [CONNECTED] wss://gateway.discord.gg/?encoding=json&v=9&compress=zlib-stream in 0 ms
web.cf975c06a65832e459b4.js:12 [GatewaySocket] [READY] took 2695ms, as 86159d095cd4c5533b31716a9cc8cc5c
web.cf975c06a65832e459b4.js:12 [Flux] Dispatching CONNECTION_OPEN
web.cf975c06a65832e459b4.js:12 [DatabaseManager] removing database (user: 239982644916518913, database: undefined)
web.cf975c06a65832e459b4.js:12 [Spotify] WS Connecting
web.cf975c06a65832e459b4.js:12 [Spotify] Added account: zany130ao
Logger.ts:40  Vencord+   PluginManager  Starting plugins (stage WebpackReady)
Logger.ts:40  Vencord+   PluginManager  Starting plugin BadgeAPI
Logger.ts:40  Vencord+   PluginManager  Starting plugin SupportHelper
Logger.ts:40  Vencord+   PluginManager  Starting plugin BetterRoleContext
Logger.ts:40  Vencord+   PluginManager  Starting plugin BetterSessions
Logger.ts:40  Vencord+   PluginManager  Starting plugin BlurNSFW
Logger.ts:40  Vencord+   PluginManager  Starting plugin ClearURLs
Logger.ts:40  Vencord+   PluginManager  Starting plugin Decor
Logger.ts:40  Vencord+   PluginManager  Starting plugin Experiments
Logger.ts:40  Vencord+   PluginManager  Starting plugin F8Break
Logger.ts:40  Vencord+   PluginManager  Starting plugin GameActivityToggle
Logger.ts:40  Vencord+   PluginManager  Starting plugin ImageZoom
Logger.ts:40  Vencord+   PluginManager  Starting plugin ImplicitRelationships
Logger.ts:40  Vencord+   PluginManager  Starting plugin MessageClickActions
Logger.ts:40  Vencord+   PluginManager  Starting plugin MessageLinkEmbeds
Logger.ts:40  Vencord+   PluginManager  Starting plugin MessageLogger
Logger.ts:40  Vencord+   PluginManager  Starting plugin MessageTags
Logger.ts:40  Vencord+   PluginManager  Starting plugin PinDMs
Logger.ts:40  Vencord+   PluginManager  Starting plugin PlatformIndicators
Logger.ts:40  Vencord+   PluginManager  Starting plugin QuickReply
Logger.ts:40  Vencord+   PluginManager  Starting plugin ReactErrorDecoder
Logger.ts:40  Vencord+   PluginManager  Starting plugin ReadAllNotificationsButton
Logger.ts:40  Vencord+   PluginManager  Starting plugin RelationshipNotifier
Logger.ts:40  Vencord+   PluginManager  Starting plugin ReviewDB
Logger.ts:40  Vencord+   PluginManager  Starting plugin Summaries
Logger.ts:40  Vencord+   PluginManager  Starting plugin ServerListIndicators
Logger.ts:40  Vencord+   PluginManager  Starting plugin ShikiCodeblocks
Logger.ts:40  Vencord+   PluginManager  Starting plugin SilentMessageToggle
Logger.ts:40  Vencord+   PluginManager  Starting plugin SpotifyControls
Logger.ts:40  Vencord+   PluginManager  Starting plugin Translate
Logger.ts:40  Vencord+   PluginManager  Starting plugin USRBG
Logger.ts:40  Vencord+   PluginManager  Starting plugin WebContextMenus
Logger.ts:40  Vencord+   PluginManager  Starting plugin WebKeybinds
Logger.ts:40  Vencord+   PluginManager  Starting plugin KeywordNotify
Logger.ts:40  Vencord+   PluginManager  Starting plugin ServerSearch
Logger.ts:40  Vencord+   PluginManager  Starting plugin ThemeLibrary
Logger.ts:40  Vencord+   PluginManager  Starting plugin AlwaysPing
Logger.ts:40  Vencord+   PluginManager  Starting plugin DecodeBase64
Logger.ts:40  Vencord+   PluginManager  Starting plugin BetterQuests
Logger.ts:40  Vencord+   PluginManager  Starting plugin BetterScreenshare
Logger.ts:40  Vencord+   PluginManager  Starting plugin BetterUserArea
Logger.ts:40  Vencord+   PluginManager  Starting plugin FavouriteMedia
Logger.ts:40  Vencord+   PluginManager  Starting plugin GlobalBadges
Logger.ts:40  Vencord+   PluginManager  Starting plugin GodMode
Logger.ts:40  Vencord+   PluginManager  Starting plugin GrammarFix
Logger.ts:40  Vencord+   PluginManager  Starting plugin Keyboard Sounds
Logger.ts:40  Vencord+   PluginManager  Starting plugin MediaDownloader
Logger.ts:40  Vencord+   PluginManager  Starting plugin QuickSearch
Logger.ts:40  Vencord+   PluginManager  Starting plugin RelationshipIndicators
Logger.ts:40  Vencord+   PluginManager  Starting plugin RelationshipPruner
Logger.ts:40  Vencord+   PluginManager  Starting plugin SearchUserMessages
Logger.ts:40  Vencord+   PluginManager  Starting plugin ShowCharacterCount
Logger.ts:40  Vencord+   PluginManager  Starting plugin UserPFP
Logger.ts:40  Vencord+   PluginManager  Starting plugin ColorMessage
web.cf975c06a65832e459b4.js:12 [discord_protos.discord_users.v1.PreloadedUserSettings] Updating appearance with delay 0
web.cf975c06a65832e459b4.js:12 [discord_protos.discord_users.v1.PreloadedUserSettings] Scheduling save from markDirty
web.cf975c06a65832e459b4.js:12 [discord_protos.discord_users.v1.PreloadedUserSettings] Persisting proto
web.cf975c06a65832e459b4.js:12 [GatewaySocket] [READY_SUPPLEMENTAL] took 3043ms
WebpackModule706898:2 undefined
/api/v9/users/@me/affinities/v2/users:1 
        
        
       Failed to load resource: the server responded with a status of 401 ()
web.cf975c06a65832e459b4.js:12 [Spotify] WS Connected
WebpackModule706898:2 undefined
web.cf975c06a65832e459b4.js:12 [Spotify] Profile updated for zany130ao: isPremium = true
web.cf975c06a65832e459b4.js:12 [Spotify] Devices updated for zany130ao: Array(0)
@me:1 Unrecognized Content-Security-Policy directive 'prefetch-src'.
Logger.ts:40  Vencord+   Cloud:Settings  Settings up to date
/api/v9/streams/guild%3A662007546982432778%3A1193595921167044770%3A543067844380721152/preview?version=1727206915581:1 
        
        
       Failed to load resource: the server responded with a status of 401 ()
3WebpackModule706898:2 undefined
web.cf975c06a65832e459b4.js:12 [ReadStateStore] Deleting ReadState 598654977702887425 0 Object
web.cf975c06a65832e459b4.js:12 [ReadStateStore] Deleting ReadState 1176891169444204574 0 Object
2WebpackModule706898:2 undefined
/api/v9/streams/guild%3A662007546982432778%3A1193595921167044770%3A543067844380721152/preview?version=1727206927919:1 
        
        
       Failed to load resource: the server responded with a status of 401 ()
3WebpackModule706898:2 undefined
/api/v9/streams/guild%3A662007546982432778%3A1193595921167044770%3A543067844380721152/preview?version=1727206948693:1 
        
        
       Failed to load resource: the server responded with a status of 401 ()
4WebpackModule706898:2 undefined
/api/v9/streams/guild%3A662007546982432778%3A1193595921167044770%3A543067844380721152/preview?version=1727206979206:1 
        
        
       Failed to load resource: the server responded with a status of 401 ()
WebpackModule706898:2 undefined
web.cf975c06a65832e459b4.js:12 [ComponentDispatchUtils] ComponentDispatch.subscribe: Attempting to add a duplicate listener POPOUT_CLOSE
(anonymous) @ web.cf975c06a65832e459b4.js:12
web.cf975c06a65832e459b4.js:12 [MessageActionCreators] Fetching messages for 853130811581530145 between undefined and undefined. jump={"jumpType":"ANIMATED"}
web.cf975c06a65832e459b4.js:12 [MessageActionCreators] Fetched 50 messages for 853130811581530145 isBefore:false isAfter:false
web.cf975c06a65832e459b4.js:12 [ChannelMessages] loadComplete: resetting state for channelId=853130811581530145, sending.length=0
2WebpackModule706898:2 undefined
/api/v9/streams/guild%3A662007546982432778%3A1193595921167044770%3A543067844380721152/preview?version=1727207021499:1 
        
        
       Failed to load resource: the server responded with a status of 401 ()
/api/v9/streams/guild%3A662007546982432778%3A1193595921167044770%3A543067844380721152/preview?version=1727207072040:1 
        
        
       Failed to load resource: the server responded with a status of 401 ()
WebpackModule706898:2 undefined
2Third-party cookie will be blocked in future Chrome versions as part of Privacy Sandbox.
web.cf975c06a65832e459b4.js:12 
        
        
       GET https://discord.com/api/v9/streams/guild%3A662007546982432778%3A1193595921167044770%3A543067844380721152/preview?version=1727207133817 401 (Unauthorized)
A._end @ web.cf975c06a65832e459b4.js:12
A.end @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
l.then @ web.cf975c06a65832e459b4.js:12
E @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
m @ web.cf975c06a65832e459b4.js:12
P @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
Promise.then (async)
getPreviewURL @ web.cf975c06a65832e459b4.js:12
(anonymous) @ c92a205811a4901e300f.js:1
l @ web.cf975c06a65832e459b4.js:12
u @ web.cf975c06a65832e459b4.js:12
v @ c92a205811a4901e300f.js:1
aA @ WebpackModule502920:2
sh @ WebpackModule502920:2
l @ WebpackModule502920:2
ok @ WebpackModule502920:2
eval @ WebpackModule502920:2
oG @ WebpackModule502920:2
oR @ WebpackModule502920:2
N @ web.cf975c06a65832e459b4.js:12
D @ web.cf975c06a65832e459b4.js:12
T.C.<computed> @ web.cf975c06a65832e459b4.js:1
L @ web.cf975c06a65832e459b4.js:1
b @ web.cf975c06a65832e459b4.js:1
web.cf975c06a65832e459b4.js:12 [MessageActionCreators] Fetching messages for 969001281765130300 between undefined and undefined. jump={"jumpType":"ANIMATED"}
web.cf975c06a65832e459b4.js:12 
        
        
       GET https://discord.com/api/v9/channels/969001281765130300/messages?limit=50 403 (Forbidden)
A._end @ web.cf975c06a65832e459b4.js:12
A.end @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
l.then @ web.cf975c06a65832e459b4.js:12
E @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
m @ web.cf975c06a65832e459b4.js:12
fetchMessages @ web.cf975c06a65832e459b4.js:12
D @ WebpackModule348245:2
B @ WebpackModule348245:2
(anonymous) @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
_dispatch @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
log @ web.cf975c06a65832e459b4.js:12
_dispatchWithLogging @ web.cf975c06a65832e459b4.js:12
_dispatchWithDevtools @ web.cf975c06a65832e459b4.js:12
dispatch_CHANNEL_PRELOAD @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
flushWaitQueue @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
dispatch @ web.cf975c06a65832e459b4.js:12
preload @ WebpackModule493683:2
_ensureCount @ OnlineMemberCountStore.ts:25
(anonymous) @ OnlineMemberCountStore.ts:32
Promise.then (async)
next @ Queue.ts:40
run @ Queue.ts:48
push @ Queue.ts:61
ensureCount @ OnlineMemberCountStore.ts:31
(anonymous) @ MemberCount.tsx:48
sF @ WebpackModule502920:2
oV @ WebpackModule502920:2
eval @ WebpackModule502920:2
oF @ WebpackModule502920:2
oL @ WebpackModule502920:2
r5 @ WebpackModule502920:2
ob @ WebpackModule502920:2
t.flushSync @ WebpackModule502920:2
toggleShow @ web.cf975c06a65832e459b4.js:12
show @ web.cf975c06a65832e459b4.js:12
(anonymous) @ web.cf975c06a65832e459b4.js:12
ek @ WebpackModule502920:2
eY @ WebpackModule502920:2
eval @ WebpackModule502920:2
re @ WebpackModule502920:2
rt @ WebpackModule502920:2
eval @ WebpackModule502920:2
oD @ WebpackModule502920:2
eU @ WebpackModule502920:2
ro @ WebpackModule502920:2
tk @ WebpackModule502920:2
tG @ WebpackModule502920:2
web.cf975c06a65832e459b4.js:12 [MessageActionCreators] Failed to fetch messages for 969001281765130300
2WebpackModule706898:2 undefined
```

### Request Agreement

- [X] I am using Discord Stable or tried on Stable and this bug happens there as well
- [X] I have read the requirements for opening an issue above

**Comment by 0xJWLabs** [comment]
I was going to create issue for this, thanks for making the issue first!

**Comment by RobinRMC** [comment]
Since the author of ChannelTabs doesn't maintain the plugin anymore, I'll make an issue about this on the repository of Equicord, because the maintainer of it maintains plugins that have been abandoned by their original authors.

**Comment by RobinRMC** [comment]
Nevermind, it seems like they have already fixed the issue. I'll update Vencord+ tomorrow.

**Comment by RobinRMC** [comment]
The issue has been fixed with commit c81897210051fc2653f85689bccdfa8b8a04dfb0.

**Comment by RobinRMC** [comment]
Multiple people have confirmed that the issue has been resolved.
